tests/test_context.py

```diff
@@ -21,7 +21,22 @@ def test_context_with_path(self):
             ignore_dependencies=sentinel.ignore_dependencies,
         )
 
-        sentinel.project_path = "project_path/to/sceptre"
+        sentinel.project_path = f"{getcwd()}/project_path/to/sceptre"
+        assert self.context.project_path.replace(path.sep, "/") == sentinel.project_path
+
+    def test_context_with_relative_path(self):
+        self.context = SceptreContext(
+            project_path="./project_path/to/sceptre",
+            command_path="command-path",
+            command_params=sentinel.command_params,
+            user_variables=sentinel.user_variables,
+            options=sentinel.options,
+            output_format=sentinel.output_format,
+            no_colour=sentinel.no_colour,
+            ignore_dependencies=sentinel.ignore_dependencies,
+        )
+
+        sentinel.project_path = f"{getcwd()}/project_path/to/sceptre"
         assert self.context.project_path.replace(path.sep, "/") == sentinel.project_path
```

Comment:
Using the sentinel here doesn't really get you anything. 
```suggestion
        expected = f"{getcwd()}/project_path/to/sceptre"
        assert self.context.project_path.replace(path.sep, "/") == expected
```